README.md

```diff
@@ -255,6 +255,25 @@ The default prompt format uses this to add spaces only if the feature would
 render. In that way the prompt always looks well spaced out no matter how many
 features are rendering.
 
+### Prefixing and Suffixing the Full git-radar Prompt
+
+Much like the previous section, it may be that we want our git-radar prompt to
```

Comment:
No need to reference to the previous section. As this section is solving a particular need it can be direct and specific about that need and how to solve it.